### PR TITLE
Deep learning terms

### PR DESCRIPTION
**Pull request by dsmits**
Added terms related to the deep learning workshop

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -10110,4 +10110,85 @@
     term: "non-parametric (statistics)"
     def: >
       A branch of statistical tests which do not assume a known distribution of the population which the samples were taken from (Kruskal-Wallis and Dunn test are examples of non-parametric tests).
-     
+- slug: artificial_intelligence
+  ref:
+    - nlp
+    - ma
```
Comment (svenvanderburg) [inline glossary.yml]:
```suggestion
    - ml
```
Did you mean machine learning?

File: glossary.yml [hunk 2]
```diff
@@ -10110,4 +10110,85 @@
...
+    determines the pace at which the network adjusts the weights to move down the loss gradient. 
+    A large learning rate can speed up training, but the network might overshoot and miss the 
+    minimum. A small learning rate will overshoot less, but will be slower. It can also get more 
+    easily stuck in local minima.
+
+- slug: class_imbalance
+  ref:
+    - machine_learning
+  def: >
+    Class imbalance refers to the problem [machine learning](machine_learning) where there is an
```
Comment (svenvanderburg) [inline glossary.yml:10182]:
```suggestion
    Class imbalance refers to the problem in [machine learning](machine_learning) where there is an 
```

**Review by svenvanderburg — APPROVED** [review]
Looks good @dsmits ! I added 2 small comments.

**Review by zkamvar — APPROVED** [review]
LGTM, The only thing left is https://github.com/carpentries/glosario/pull/584#discussion_r1196387875, which needs to be addressed by @dsmits because the PR does not enable edits from maintainers.

Once that's resolved we'll merge

**Comment by elletjies** [comment]
@all-contributors please add @dsmits for translation

**Pull request merged**